cdk/amplify-production-stack.ts

```diff
@@ -76,6 +76,9 @@ export class AmplifyProductionStack extends cdk.Stack {
       { source: '/textpresso/fb/',                     target: 'https://www.alliancegenome.org/textpresso/fb/tpc',                                    status: amplify.RedirectStatus.PERMANENT_REDIRECT },
       { source: '/textpresso/fb/<*>',                  target: 'https://fb-textpresso.alliancegenome.org/<*>',                                        status: amplify.RedirectStatus.REWRITE },
 
+      { source: '/blast/',                             target: 'https://www.alliancegenome.org/blast',                                              status: amplify.RedirectStatus.PERMANENT_REDIRECT },
```

Comment:
This link needs to have the slash and non slash version. See the way the swagger-ui redirects are done.